mezzo/src/pages/charts/charts.js

```diff
@@ -37,17 +118,72 @@ const Charts = () => {
     togglePlayer(newPlayerInfo);
   }
 
+  function likeSong(value, song) {
```

Comment:
Can you make likeSong into a export component in the component folder so we can reuse it in our components

Reply:
likeSong is not a component, just a function that calls our end point though `likeSongPost.js`.